app/assets/stylesheets/common/base/discourse.scss

```diff
@@ -649,8 +649,22 @@ table {
 
 // Special elements
 
+#main-outlet-wrapper {
+  display: grid;
+  grid-template-areas: "content";
+  grid-template-columns: 1fr;
+  gap: 0;
+
+  &.has-sidebar {
+    grid-template-areas: "sidebar content";
+    grid-template-columns: 240px 1fr;
```

Comment:
This will look odd on mobile, maybe we should move the grid layout to a desktop stylesheet for now.

Reply:
Good point, I've moved it over to the desktop stylesheet so at least the damage is limited to desktop.